flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/operators/multipleinput/MultipleInputStreamOperatorBase.java

```diff
@@ -245,9 +245,9 @@ private void createAllOperators(StreamOperatorParameters<RowData> parameters) {
                     output = outputs[0];
                 } else {
                     if (isObjectReuseEnabled) {
```

Comment:
please add some comments for this behavior, just like the comments in OperatorChain#createOutputCollector method